src/types/User.ts

```diff
@@ -0,0 +1,11 @@
+import type { Writable } from './Writable';
+
+interface User {
+	readonly id: number;
+	subject: string;
```

Comment:
Just for my own understanding, the "subject" of the User schema is really the "subject" of the JWT (which is in turn what refers to the 'User' of the JWT)?

Reply:
Correct -- and to that point, I wonder if `subject` should actually be used here since it's removed from the jwt context.  Maybe it should be `authenticationSubject` or something entirely different!

Reply:
dan you took advice from a robot
dan you owe them now
dan
dan be careful who your friends are